docs/development/setup-local.md

```diff
@@ -0,0 +1,254 @@
+
+# attune local dev setup
+
+This document walks though setting up Attune to run locally outside of Docker,
+and then testing that it is working end-to-end using a Docker container.
+
+Each sub-heading is meant to be read as an ordered step.
+
+## Clone Attune
+
+Clone the main repository, which you've probably already done since you're reading this:
+```sh
+; git clone git@github.com:attunehq/attune.git
+```
+
+If you're not an employee, you probably want to delete the `packages/attune-ee` folder if it exists
+so that Cargo can properly build:
+```sh
+; rm -rf packages/attune-ee
+```
+
+If you're an employee, you'll also want to clone the git submodules:
+```sh
+; git submodule init
+; git submodule update
+```
+
+## Install prerequisites
+
+You'll need the following tools installed and configured:
+- Docker ([desktop](https://docs.docker.com/get-started/get-docker/), [engine](https://docs.docker.com/engine/install/))
+- PostgreSQL ([Docker](https://hub.docker.com/_/postgres), [macOS](https://postgresapp.com/))
+- Minio ([quickstart](https://charts.min.io/))
```

Comment:
You can spin these two up in Docker instead, and that's how they're meant to be used.

Reply:
Thinking more I'll just remove these, I agree we should prefer simplicity actually